experiment/methods/BSR.py

```diff
@@ -0,0 +1,20 @@
+
+import bsr
+from bsr.bsr_class import BSR
+
+
+# number of simple signals
+K = 3
+MM = 50
+
+hyper_params = [{'treeNum': 3, 'itrNum':50, 'alpha1':0.4, 'alpha2':0.4, 'beta':-1}]
```

Comment:
please specify the hyperparam values as lists, like in the `param_grid` variable in [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html). We still need to work out fixed numbers of evaluations for each method, but start with something reasonable.

Reply:
dev is merged now, so no need to rebase